fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -2702,7 +2720,7 @@ private void sendFields(List<String> colNames, List<Type> types) throws IOExcept
         // send field one by one
         for (int i = 0; i < colNames.size(); ++i) {
             serializer.reset();
-            if (prepareStmt != null && isExecuteStmt) {
+            if (prepareStmt != null && isExecuteStmt && prepareStmt.isPointQuery()) {
```

Comment:
sendFields should not use prepareStmt.isPointQuery() it has not relation to this function